_posts/en/newsletters/2023-01-25-newsletter.md

```diff
@@ -0,0 +1,121 @@
+    significant advantage of being much easier to get into
+    production than the `SIGHASH_GROUP` soft fork consensus change.
+
+- **Request for proof that an async payment was accepted:** Valentine
+  Wallace [posted][wallace pop] to the Lightning-Dev mailing list a
+  request for researchers to investigate how someone making an [async
+  payment][topic async payments] could receive proof that they paid.
+  For traditional LN payments, the receiver generates a secret which
+  gets digested by a hash function; that digest is given to the spender
+  in a signed invoice; the spender uses an [HTLC][topic htlc] to pay
+  anyone who discloses the original secret.  That disclosed secret
+  proves the spender paid the digest contained in the signed invoice.
+
+    By contrast, async payments are accepted when the receiver is
+    offline, so they can't reveal a secret, preventing the creation of a
+    proof of payment in the current LN model.  Wallace asks researchers
+    to consider investigating how proof of payment for async payments
+    could be obtained, either in LN's current HTLC-based system or a
+    future upgrade to [PTLCs][topic ptlc].
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #26325][] improves the results of the `scanblocks` rpc
+  by removing false positives in a second pass. `scanblock` may be used
+  to find blocks transactions relevant to a provided set of descriptors.
+  Since scanning against the filters may falsely indicate some blocks
+  that do not actually contain relevant transactions, this PR validates
+  each hit in another pass to see whether blocks actually correspond to
+  the passed descriptors before providing the results to the caller.
```

Comment:
It may be worth mentioning that this functionality is disabled by default (for performance reasons); the user must specify `-filter_false_positives` to enable it.